KarhooUISDK/Screens/CheckoutScreen/Checkout/KarhooCheckoutPresenter.swift

```diff
@@ -92,6 +92,10 @@ final class KarhooCheckoutPresenter: CheckoutPresenter {
         view.set(quote: quote, showLoyalty: showLoyalty, loyaltyId: loyaltyId)
     }
 
+    func screenWillAppear() {
+        analyticsScreenOpened()
```

Comment:
all analytics event have "report" prefix: reportBookingEvent, reportPaymentSuccess etc. Maybe you should name this function in the same way?